test_requirements.txt

```diff
@@ -1,8 +1,8 @@
-flake8==2.6.0
-pep8==1.7.0
-pyflakes==1.0.0
-flake8-print==2.0.2
-responses==0.7.0
-pytest==3.2.3
-pytest-cov==2.5.1
-pytest-html==1.8.0
+flake8
```

Comment:
It might be worth changing these to e.g.
```
flake8~=2.6
```
This would allow upgrading minor and patch versions but avoid changing major versions.

Reply:
It depends how we want to look at it, either we can keep them at the latest or we'd have to update them regularly, I'm not too particular either way tbh

Reply:
Personally, I much prefer pinning to major versions.

If a major version of a test dependency changes then it will most likely cause issues with existing code (since the API has changed). That tends to lead to inscrutable errors and it can take a while to realise that it's caused by a test dependency being updated.

I realise this isn't a big issue, so happy to approve if you don't want to go this route.